nautobot/extras/models/statuses.py

```diff
@@ -33,6 +35,13 @@ class Meta:
         ordering = ["name"]
         verbose_name_plural = "statuses"
 
+    def get_color_display(self):
```

Comment:
Would it make sense to add this to the base `NameColorContentTypesModel` class instead of only to `Status`?